ui/page/components/components.go

```diff
@@ -111,82 +111,93 @@ func UniformHorizontalPadding(gtx layout.Context, body layout.Widget) layout.Dim
-			} else if wal.TxMatchesFilter(tx, dcrlibwallet.TxFilterImmature) {
-				txStatus.Title = values.String(values.StrImmature)
-				txStatus.Icon = l.Theme.Icons.TicketImmatureIcon
-				txStatus.Color = l.Theme.Color.LightBlue6
-				txStatus.TicketStatus = dcrlibwallet.TicketStatusImmature
-				txStatus.ProgressBarColor = l.Theme.Color.LightBlue5
-				txStatus.ProgressTrackColor = l.Theme.Color.LightBlue3
-				txStatus.Background = l.Theme.Color.LightBlue
-			} else if ticketSpender != nil {
-				if ticketSpender.Type == dcrlibwallet.TxTypeVote {
-					txStatus.Title = values.String(values.StrVoted)
-					txStatus.Icon = l.Theme.Icons.TicketVotedIcon
-					txStatus.Color = l.Theme.Color.Turquoise700
-					txStatus.TicketStatus = dcrlibwallet.TicketStatusVotedOrRevoked
-					txStatus.ProgressBarColor = l.Theme.Color.Turquoise300
-					txStatus.ProgressTrackColor = l.Theme.Color.Turquoise100
-					txStatus.Background = l.Theme.Color.Success2
+	switch tx.Direction {
+	case dcrlibwallet.TxDirectionSent:
+		txStatus.Title = "Sent"
+		txStatus.Icon = l.Theme.Icons.SendIcon
+	case dcrlibwallet.TxDirectionReceived:
+		txStatus.Title = "Received"
+		txStatus.Icon = l.Theme.Icons.ReceiveIcon
+	default:
+		txStatus.Title = "Transferred"
+		txStatus.Icon = l.Theme.Icons.Transferred
+	}
+
+	// replace icon for staking tx types
+	if wal.TxMatchesFilter(tx, dcrlibwallet.TxFilterStaking) {
+		switch tx.Type {
+		case dcrlibwallet.TxTypeTicketPurchase:
+			{
+				if wal.TxMatchesFilter(tx, dcrlibwallet.TxFilterUnmined) {
+					txStatus.Title = "Unmined"
+					txStatus.Icon = l.Theme.Icons.TicketUnminedIcon
+					txStatus.TicketStatus = dcrlibwallet.TicketStatusUnmined
+					txStatus.Color = l.Theme.Color.LightBlue6
+					txStatus.Background = l.Theme.Color.LightBlue
+				} else if wal.TxMatchesFilter(tx, dcrlibwallet.TxFilterImmature) {
+					txStatus.Title = "Immature"
+					txStatus.Icon = l.Theme.Icons.TicketImmatureIcon
+					txStatus.Color = l.Theme.Color.LightBlue6
+					txStatus.TicketStatus = dcrlibwallet.TicketStatusImmature
+					txStatus.ProgressBarColor = l.Theme.Color.LightBlue5
+					txStatus.ProgressTrackColor = l.Theme.Color.LightBlue3
+					txStatus.Background = l.Theme.Color.LightBlue
+				} else if wal.TxMatchesFilter(tx, dcrlibwallet.TxFilterLive) {
+					txStatus.Title = "Live"
+					txStatus.Icon = l.Theme.Icons.TicketLiveIcon
+					txStatus.Color = l.Theme.Color.Primary
+					txStatus.TicketStatus = dcrlibwallet.TicketStatusLive
+					txStatus.ProgressBarColor = l.Theme.Color.Primary
+					txStatus.ProgressTrackColor = l.Theme.Color.LightBlue4
+					txStatus.Background = l.Theme.Color.Primary50
+				} else if wal.TxMatchesFilter(tx, dcrlibwallet.TxFilterExpired) {
+					txStatus.Title = "Expired"
+					txStatus.Icon = l.Theme.Icons.TicketExpiredIcon
+					txStatus.Color = l.Theme.Color.GrayText2
+					txStatus.TicketStatus = dcrlibwallet.TicketStatusExpired
+					txStatus.Background = l.Theme.Color.Gray4
```

Comment:
```suggestion
	case dcrlibwallet.TxDirectionSent:
		txStatus.Title = values.String(values.StrSent)
		txStatus.Icon = l.Theme.Icons.SendIcon
	case dcrlibwallet.TxDirectionReceived:
		txStatus.Title = values.String(values.StrReceived)
		
```

Lets localize all string variables as they where initially.